src/idl_gen_general.cpp

```diff
@@ -89,6 +89,7 @@ struct LanguageParameters {
   const char *namespace_ident;
   const char *namespace_begin;
   const char *namespace_end;
+  const char *namespace_separator;
```

Comment:
Not a big deal either way. Could be useful in the future, then again the only language I know that doesn't use "." is C++.